robots/preservation_ingest/update_catalog.rb

```diff
@@ -20,8 +24,34 @@ def perform(druid)
 
         private
 
+        def http_args
+          {
+            druid: druid,
+            incoming_version: moab_object.current_version_id,
+            incoming_size: moab_object.size,
+            storage_location: moab_object.storage_root
+          }
+        end
+
+        def handler
+          proc do |exception, attempt_number, _total_delay|
+            LyberCore::Log.warn("Try #{attempt_number} failed: #{exception.message}")
+            raise exception if attempt_number == 3
```

Comment:
now that I'm trolling the actual production logs, it is helpful if the druid (and possibly other context) are mentioned when an exception is thrown -- might we be able to add them here?  Could be a separate ticket.  Not urgent.